style.css

```diff
@@ -0,0 +1,247 @@
+* {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.bgc {
+  background-image: url(./images/f-bg.png);
+  background-repeat: no-repeat;
+  background-size: contain;
+  height: 957px;
+  background-position: 0 -40px;
+  position: relative;
+}
+
+body {
+  background-color: #191765;
+  color: #fff;
+  width: 375px;
+}
+
+.soft {
+  font-family: 'Roboto', sans-serif;
+  font-weight: 500;
```

Comment:
- [ ] [Optional] When adding margins, width, height to elements on your page, always favor percent base units (%, vh, vw) over fixed units (px). This will help improve the responsiveness of your page